src/main/java/com/github/instagram4j/instagram4j/actions/status/StatusAction.java

```diff
@@ -25,7 +25,11 @@
      * @return IGResponse
      */
     public CompletableFuture<IGResponse> setStatus(String text, String emoji, long expires_at, boolean should_notify, String status_type) {
+<<<<<<< HEAD
```

Comment:
There seems to be unfinished merge conflict. Please resolve by only keeping the HEAD version. (The one marked as <<<<<<< HEAD and ending with =======)

Reply:
fixed